monitoring/mock_uss/interaction_logging/interactions.py

```diff
@@ -0,0 +1,28 @@
+from enum import Enum
+from typing import List
+
+from implicitdict import ImplicitDict
+import yaml
+from yaml.representer import Representer
+
+from monitoring.monitorlib.fetch import Query
+
+
+class QueryDirection(str, Enum):
+    Incoming = "Incoming"
+    """The query originated from a remote client and was handled by the system reporting the interaction."""
+
+    Outgoing = "Outgoing"
+    """The system reporting the interaction initiated the query to a remote server."""
+
+
+class Interaction(ImplicitDict):
+    query: Query
+    direction: QueryDirection
+
+
+yaml.add_representer(Interaction, Representer.represent_dict)
+
+
+class ListLogsResponse(ImplicitDict):
```

Comment:
Same as above, should it not be defined in monitorlib, as it will be used by uss_qualifier?

Reply:
I am good if we can import from mock_uss into uss_qualifier.